acsf_tools.drush.inc

```diff
@@ -213,6 +227,57 @@ function drush_acsf_tools_dump() {
   }
 }
 
+/**
+ * Action callback for acsf-tools-restore command.
+ */
+function drush_acsf_tools_restore() {
+  // Ask for confirmation before running the command.
+  if (!_drush_acsf_tools_prompt_confirm()) {
+    return;
+  }
+
+  // Identify source folder.
+  $source_folder = drush_get_option('source-folder');
+  if (!isset($source_folder)) {
+    $source_folder = '~/drush-backups';
+  }
+
+  if (!is_dir($source_folder)) {
+    // Source folder does not exist..
+    drush_log("\nSource folder $source_folder does not exist.", 'error');
+    return;
+  }
+
+  // Look for list of sites and loop over it.
+  if ($sites = _drush_acsf_tools_get_sites()) {
+    $arguments = drush_get_arguments();
+
+    $options = drush_get_context('cli');
+    unset($options['php']);
+    unset($options['php-options']);
+    unset($options['source-folder']);
+
+    foreach ($sites as $details) {
+      $domain = $details['domains'][0];
+      $prefix = explode('.', $domain)[0];
+
+      $source_file = $source_folder . '/' . $prefix . '.sql';
```

Comment:
The files are likely to have been gzipped (from the dump command)

Reply:
Will add --gzip argument support.

Reply:
@sdelbosc I added --gzip option.